### PR TITLE
Instructions for serving locally and contributing

### PR DESCRIPTION
**Pull request by tahirozdemir34**
Related issue is #4

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,4 +12,58 @@ If you encounter any issues with these themes on any devices, please submit a bu
...
+
+In order deploy the website locally we need Ruby and some "gems" installed on our machines.
+
+#### 3.1. Windows
+
+1. Download and Install a Ruby+Devkit version from [RubyInstaller Downloads](https://rubyinstaller.org/downloads/). Use default options for installation.
+2. Run the `ridk install` step on the last stage of the installation wizard. This is needed for installing gems with native extensions. You can find additional information regarding this in the [RubyInstaller Documentation](https://github.com/oneclick/rubyinstaller2#using-the-installer-on-a-target-system).
+3. Open a new command prompt window from the start menu, so that changes to the PATH environment variable becomes effective. Install Jekyll via `gem install jekyll` and for compatibility install Bundler v1.12 via `gem install bundler -v 1.12`
+4. Check if Jekyll and Bundler installed properly: `jekyll -v` and `bundler -v`
+5. Go to directiory that you clone the your forked repository and run `jekyll serve`. In browser open "127.0.0.1:4000".
```
Comment (kbsezginel) [inline CONTRIBUTING.md]: 5. Go to the directory of your forked repository and run `jekyll serve`. In your browser open `127.0.0.1:4000`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,4 +12,58 @@ If you encounter any issues with these themes on any devices, please submit a bu
...
+### 3. Install Dependencies to Serve Website Locally [*]
+
+In order deploy the website locally we need Ruby and some "gems" installed on our machines.
+
+#### 3.1. Windows
+
+1. Download and Install a Ruby+Devkit version from [RubyInstaller Downloads](https://rubyinstaller.org/downloads/). Use default options for installation.
+2. Run the `ridk install` step on the last stage of the installation wizard. This is needed for installing gems with native extensions. You can find additional information regarding this in the [RubyInstaller Documentation](https://github.com/oneclick/rubyinstaller2#using-the-installer-on-a-target-system).
+3. Open a new command prompt window from the start menu, so that changes to the PATH environment variable becomes effective. Install Jekyll via `gem install jekyll` and for compatibility install Bundler v1.12 via `gem install bundler -v 1.12`
+4. Check if Jekyll and Bundler installed properly: `jekyll -v` and `bundler -v`
```
Comment (kbsezginel) [inline CONTRIBUTING.md]: 4. Check if Jekyll and Bundler are installed properly: `jekyll -v` and `bundler -v`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,4 +12,58 @@ If you encounter any issues with these themes on any devices, please submit a bu
...
+#### 3.3. Ubuntu
+
+1. Run `sudo apt-get install ruby-full build-essential zlib1g-dev`
+2. To avoid install gems as root user add following lines to your ".bashrc" or related configuration file that is used by your shell (for example ".zshrc" if tou use zsh). This lines tells your system to install gem files to "~/gems"
+```
+export GEM_HOME="$HOME/gems"
+export PATH="$HOME/gems/bin:$PATH"
+```
+3. Install Jekyll via `gem install jekyll` and for compatibility install Bundler v1.12 via `gem install bundler -v 1.12`
+4. Go to directiory that you clone the your forked repository and run `jekyll serve`. In browser open "127.0.0.1:4000".
```
Comment (kbsezginel) [inline CONTRIBUTING.md]: 4. Go to the directory of your forked repository and run `jekyll serve`. In your browser open `127.0.0.1:4000`.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -12,4 +12,58 @@ If you encounter any issues with these themes on any devices, please submit a bu
...
+
+## 4. Make the changes you want
```
Comment (kbsezginel) [inline CONTRIBUTING.md]: Should be 5.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -12,4 +12,58 @@ If you encounter any issues with these themes on any devices, please submit a bu
...
+## 5. Go to GitHub page of your forked repo and create a pull request
```
Comment (kbsezginel) [inline CONTRIBUTING.md]: Should be 6.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -12,4 +12,58 @@ If you encounter any issues with these themes on any devices, please submit a bu
...
+export PATH="$HOME/gems/bin:$PATH"
+```
+3. Install Jekyll via `gem install jekyll`
+* If you get an error that says
+`gcc: error: /usr/lib/rpm/redhat/redhat-hardened-cc1: No such file or directory`
+Run : `sudo dnf install redhat-rpm-config`
+* If you get an error that says `make: g++: Command not found`
+Run: `dnf install gcc-c++`
+4. Install Bundler v1.12 for compatibility via `gem install bundler -v 1.12`
+5. Go to directiory that you clone the your forked repository and run `jekyll serve`. In browser open "127.0.0.1:4000".
```
Comment (kbsezginel) [inline CONTRIBUTING.md]: 5. Go to the directory of your forked repository and run `jekyll serve`. In your browser open `127.0.0.1:4000`.

**Review by kbsezginel — CHANGES_REQUESTED** [review]
Thank you for the PR and testing all the installation instructions! I think it looks good just a couple of typos. We can merge once you fix those.

Also, since you are part of the team now you can directly commit to the develop branch of `formulanano/formulanano.github.io` and make a PR from the develop branch to the master branch next time.

File: CONTRIBUTING.md (hunk 5 above)
Comment (tahirozdemir34) [reply CONTRIBUTING.md]: Actually numbers are correct. The third one is "### 3. Install Dependencies to Serve Website Locally" and these are 4 and 5. Did I miss something here?

Comment (kbsezginel) [reply CONTRIBUTING.md]: My bad, I was confused by the inner and outer list numbers.

**Review by kbsezginel — APPROVED** [review]

**Pull request merged**